framework/src/org/checkerframework/common/value/ValueCheckerUtils.java

```diff
@@ -97,7 +98,8 @@
         } else if (AnnotationUtils.areSameByClass(anno, BottomVal.class)) {
             values = convertBottomVal(anno, castType);
         } else if (AnnotationUtils.areSameByClass(anno, UnknownVal.class)
-                || AnnotationUtils.areSameByClass(anno, ArrayLen.class)) {
+                || AnnotationUtils.areSameByClass(anno, ArrayLen.class)
+                || AnnotationUtils.areSameByClass(anno, IntRange.class)) {
```

Comment:
This sounds good.  Let's try without the conversion.  I suspect that the analysis will be precise enough, but if I am wrong we can change it in the future.  Thanks!